### ISSUE TITLE
raml2html cannot find templates outside of the current working directory (or subdirectories)

### ISSUE THREAD

**Issue by bddenhartog** [raml2html/raml2html#343]
#### Original conversation at blzjns/vscode-raml#7
---

Given the following directory structure:

```text
C:/
|
|- Users/
|  |- MyUser/
|  |  |- myTemplate.nunjucks
|  |  |- api.raml
|
|- SomeDir/
```

the following command executes successfully from `C:\`, `C:\Users`, and `C:\Users\MyUser`:

```shell
$ raml2html --template C:\Users\MyUser\myTemplate.nunjucks C:\Users\MyUser\api.raml
```

However, the above command **fails to find the template** if the user is in any other subdirectory of `C:\`, such as `C:\SomeDir\`:

```shell
$ pwd
> Path
> ----
> C:\SomeDir

$ raml2html --template C:\Users\MyUser\myTemplate.nunjucks C:\Users\MyUser\api.raml
> template not found: C:\Users\MyUser\myTemplate.nunjucks
```

Edit:

It should be noted that the command also fails to find the template if the template exists in a parent directory of my current working directory -- e.g., it would fail if I was in `C:\Users\MyUser\foo\` and was referencing the template file in `C:\Users\MyUser\`.

The failure occurs on Windows 10. I have not personally tested this on Linux.

**Comment by kevinrenskers** [comment]
Thanks for the report. The proper fix should be on the develop branch now, but it's only tested on macOS. Could you give it a spin?

**Comment by bddenhartog** [comment]
@kevinrenskers Seems to work just fine with efad9a4. Thanks for the quick fix!

**Comment by kevinrenskers** [comment]
Release cycle is simply "release when it's done and tested" :)
So now that you confirmed the fix, I published 6.2.0.
